### ISSUE TITLE
Incorporate Favorite Users

### ISSUE THREAD

**Issue by Johnjr08** [tomlandia/fuckr#141]
In the official Grindr Android/iOS apps, when viewing a profile there is the option to click the star icon to save that profile to your favorites. This makes it so you can view that profile even after they have been offline for over an hour (which makes the profile disappear from the normal grid).

It seems like fuckr has ignored the favorite users feature, and Tom stated in a previous thread that he finds favorites "useless" and doesn't want to "bloat the app" with that. 
_Source:_ https://github.com/tomlandia/fuckr/issues/36

I would personally find this feature useful, as I have some favorites saved on my grindr account. On fuckr I have to find these profiles all over again because there is no way to view my favorites. 

Any way you might reconsider your decision to leave out this feature?

**Comment by Kav28** [comment]
you can already view your favorites but you cant add to the favorite list -
ony that is not in fuckr

**Comment by RobbieTechie** [comment]
You can't even view them in the version Thom released, I added a checkbox at one point to view favorites, but it was never anything that was officially released.  I have it in my personal version too, but it was never incorporated.

**Comment by Johnjr00** [comment]
@RobbieTechie So your version currently has the ability to view favorites? If its not if the latest version you released, which version is it in?

Thanks

**Comment by RobbieTechie** [comment]
@Johnjr00 Yes what I'm using has that ability.  I don't have it in my forked version though.  I just deleted and reforked Fuckr to make sure everything was on the same page to cut down on things being inconsistent.  I can put it on google drive and put a link to it here shortly.

**Comment by Johnjr00** [comment]
@RobbieTechie Yeah if you could upload it, that would be awesome!
Much appreciated!

**Comment by RobbieTechie** [comment]
Windows 64 bit version, with favorites check box...
https://drive.google.com/open?id=0BxSmknffFGTnVG1pNnZnT1NBZ0U

**Comment by northstar77** [comment]
Hey @RobbieTechie, on a Mac here. Exactly which files from your fork incorporate the "favorites" view? So I can go in and update those manually in my files. Thanks, dude!

**Comment by RobbieTechie** [comment]
I didn't put that in my fork,  It is inside that version I linked from Google drive, and the file you are asking about is inside the /views folder, and the file itself is profiles.html
